Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -40,10 +40,13 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
         .unreadCountView.init()
         .withoutAutoresizingMaskConstraints
 
+    /*
+        TODO: ReadStatusView, Missing LLC API
     /// The view showing indicator for read status of the last message in channel.
     open private(set) lazy var readStatusView: _ChatChannelReadStatusCheckmarkView<ExtraData> = uiConfigSubviews
         .readStatusView.init()
         .withoutAutoresizingMaskConstraints
+     */
```

Comment:
hmmph I always follow rule if you have need to comment something, you should delete it. Unless it's some gaming in your WIP commit whatever... I think let's leave it there and add documentation that it is not currently supported?

Reply:
The reason I did not delete it, is because the ReadStatusView already exists. If I remove the comment, I'm afraid, the person developing this, will not use that view, because it won't know the view already exists. So I think for now this is the best approach I guess